### PR TITLE
#154 (https://github.com/masesgroup/KNet/issues/154#issuecomment-1328076661): first basic implementation

### LINKED ISSUE #110: Website is not aligned to the last repository state
**Is your feature request related to a problem? Please describe.**
Looking at https://masesgroup.github.io/KNet/: it is not aligned with https://github.com/masesgroup/KNet/blob/master/README.md and there are some typos:

image

**Describe the solution you'd like**
Align documentation and correct typos

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by masesdevelopers**
## Description

First implementation of issue #154: see https://github.com/masesgroup/KNet/issues/154#issuecomment-1328076661

## Related Issue

#154 
Fix #110 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/net/KNet/KNetCore.cs [hunk 1]
```diff
@@ -75,6 +75,24 @@ public override IEnumerable<IArgumentMetadata> CommandLineArguments
                         Default = Const.DefaultLogPath,
                         Help = "The path where log will be stored.",
                     },
+                    new ArgumentMetadata<string>()
+                    {
+                        Name = CLIParam.DisableJMX,
+                        Type = ArgumentType.Single,
+                        Help = "Disable JMX. Default is true.",
```
Comment (mariomastrodicasa) [inline src/net/KNet/KNetCore.cs:82]: Not clear, it is a switch without values

File: src/net/KNet/KNetCore.cs [hunk 2]
```diff
@@ -75,6 +75,24 @@ public override IEnumerable<IArgumentMetadata> CommandLineArguments
...
+                    },
+                    new ArgumentMetadata<string>()
+                    {
+                        Name = CLIParam.EnableJMXAuth,
+                        Type = ArgumentType.Single,
+                        Help = "Enable authenticate on JMX. Default is false",
```
Comment (mariomastrodicasa) [inline src/net/KNet/KNetCore.cs:88]: Not clear, it is a switch without values

File: src/net/KNet/KNetCore.cs [hunk 3]
```diff
@@ -75,6 +75,24 @@ public override IEnumerable<IArgumentMetadata> CommandLineArguments
...
+                    },
+                    new ArgumentMetadata<string>()
+                    {
+                        Name = CLIParam.EnableJMXSSL,
+                        Type = ArgumentType.Single,
+                        Help = "Enable SSL on JMX. Default is false.",
```
Comment (mariomastrodicasa) [inline src/net/KNet/KNetCore.cs:94]: Not clear, it is a switch without values

**Review by mariomastrodicasa — CHANGES_REQUESTED** [review]
@masesdevelopers documentation must be changed accordingly and KNetPS seems not manage the new switches.

File: src/net/KNet/KNetCore.cs (hunk 1 above)
Comment (masesdevelopers) [reply src/net/KNet/KNetCore.cs]: Changed

File: src/net/KNet/KNetCore.cs (hunk 2 above)
Comment (masesdevelopers) [reply src/net/KNet/KNetCore.cs]: Hided for now

File: src/net/KNet/KNetCore.cs (hunk 3 above)
Comment (masesdevelopers) [reply src/net/KNet/KNetCore.cs]: Hided for now

**Review by mariomastrodicasa — APPROVED** [review]

**Pull request merged**